### PR TITLE
added: output format

### PR DESCRIPTION
**Pull request by fleeto**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -33,16 +33,18 @@ func init() {
...
 			kubeImage := kubeimage.NewKubeImage(regx, allNamespace, namespace, columns, kubeconfig)
-			kubeImage.Render()
+			kubeImage.Render(format)
 		},
 	}
 	rootCmd.Flags().BoolP("all-namespaces", "A", false, "if present, list images in all namespaces.")
 	rootCmd.Flags().StringP("namespace", "n", "", "if present, list images in the specified namespace only. Use current namespace as fallback.")
 	rootCmd.Flags().StringP("columns", "c", "1,2,3", "specify the columns to display, separated by comma. [0:Namespace, 1:PodName, 2:ContainerName, 3:ContainerImage]")
 	rootCmd.Flags().StringP("kubeconfig", "k", "", "path to the kubeconfig file to use for CLI requests.")
+	rootCmd.Flags().StringP("output-format","o", "table", "output format(json/table/yaml), default value is table")
```
Comment (chenjiandongx) [inline cmd/main.go:47]: This PR doesn't provide the `yaml` format output

File: kubectl_images.go [hunk 2]
```diff
@@ -195,23 +195,47 @@ func (ki *KubeImage) summary() {
...
+			}
+			var rec []PodRecord
+			for _, v := range ki.entities {
+				rec = append(rec,
+					PodRecord{v.Namespace, v.PodName,
+						v.ContainerName, v.ContainerImage})
+			}
+			output, err := json.Marshal(rec)
+			if err != nil {
+				fmt.Println("json err:", err)
```
Comment (chenjiandongx) [inline kubectl_images.go:236]: Should we return here?

File: cmd/main.go (hunk 1 above)
Comment (fleeto) [reply cmd/main.go]: removed :)

**Review by chenjiandongx — APPROVED** [review]

**Comment by chenjiandongx** [comment]
LGTM

**Comment by chenjiandongx** [comment]
https://github.com/kubernetes-sigs/krew-index/pull/959 @fleeto I have sent new PR to the office index repo

**Pull request merged**